src/components/ClockDigit.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <span class="clock-digit_wrapper">
+        <span class="clock-digit_placeholder">8</span>
+
+        <span class="clock-digit_time">{{ digit }}</span>
+    </span>
+</template>
+
+<script>
+export default {
+    name: 'ClockDigit',
+
+    props: {
+        digit: {
+            type: String,
+            required: true,
+        },
+    },
+
+    computed: {
+        first() {
+            console.log('this.digits: ', this.digits)
+            return this.digits.split()[0]
+        },
+
+        second() {
+            return this.digits.split()[1]
+        },
+    },
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Wenn wir mit `scoped` arbeiten sollten wir das für alle Components / Modules etc in Betracht ziehen.

Reply:
Kommt immer drauf an. Würde ich definitiv so machen, aber manchmal kann es ja gewollt sein nicht zu scopen. Und wichtig ist dabei immer zu beachten, dass die Styles auf alles Kind-Komponenten vererbt werden ... ich bin ja sowieso für TailwindCSS 😀